webpack.config.js

```diff
@@ -8,6 +8,7 @@ const ExtractTextPlugin = require( 'extract-text-webpack-plugin' );
 
 const config = {
 	entry: {
+		polyfill: 'babel-polyfill',
```

Comment:
I wonder if the `useBuiltIns` works if we include the polyfill like this. Since it's a babel transformation, I think the answer is "no". but I'm not certain.